tests/docker/test-docker-compose.yml

```diff
@@ -40,16 +40,23 @@ services:
       localnet:
         ipv4_address: 192.167.10.20
 
-  # TODO - sequencer-relayer
   geth0:
     container_name: geth0
+    env_file:
+      - ./.env
     depends_on:
       - bridge0
     # FIXME - use a tagged version eventually
-    image: "ghcr.io/astriaorg/go-ethereum:latest"
+#    image: "ghcr.io/astriaorg/go-ethereum:latest"
+#    image: "ghcr.io/astriaorg/go-ethereum:local"
+    image: "ghcr.io/astriaorg/go-ethereum:custom-mempool"
```

Comment:
should we try to get this merged >.< can always update after lol

Reply:
I'm down. It just needs an approval!